### PR TITLE
Profile design

### LINKED ISSUE #425: profile page design

### PR DESCRIPTION
**Pull request by JeanVegaD**
closes #425 

# What does this PR do?
- update donor profile design 
- update sponsor profile design 
- update lifebank profile design 

# How this should be tested?
- go to http://localhost:3000
- login with a user
- go to profile section

### CODE REVIEW

**Review by AngeloCG97 — APPROVED** [review]

**Comment by AngeloCG97** [comment]
image

**Review by AngeloCG97 — COMMENTED** [review]
review details

**Review by AngeloCG97 — CHANGES_REQUESTED** [review]
Missing the field to url lifebank

File: webapp/src/routes/Profile/ProfilePageLifebank.js [hunk 1]
```diff
@@ -87,6 +211,14 @@ const ProfilePageLifebank = ({ profile }) => {
     skip: true
   })
 
+  const handleNext = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep + 1)
+  }
+
+  const handleBack = () => {
```
Comment (JustinCast) [inline webapp/src/routes/Profile/ProfilePageLifebank.js:218]: Unecessary curly braces

File: webapp/src/routes/Profile/ProfilePageLifebank.js [hunk 2]
```diff
@@ -87,6 +211,14 @@ const ProfilePageLifebank = ({ profile }) => {
     skip: true
   })
 
+  const handleNext = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep + 1)
+  }
```
Comment (JustinCast) [inline webapp/src/routes/Profile/ProfilePageLifebank.js:216]: Unnecessary curly braces

**Review by JustinCast — CHANGES_REQUESTED** [review]

**Review by AngeloCG97 — APPROVED** [review]

**Review by JustinCast — APPROVED** [review]

**Pull request merged**